addons/nightvision/functions/fnc_setupDisplayEffects.sqf

```diff
@@ -51,7 +51,11 @@ if (GVAR(fogScaling) > 0) then {
 // Note: Using BIS_fnc_rscLayer because of bug with string syntax - https://feedback.bistudio.com/T120768
 (QGVAR(display) call BIS_fnc_rscLayer) cutText ["", "PLAIN"]; // Cleanup Old Display
 if (_activated) then { // Create New Display
-    (QGVAR(display) call BIS_fnc_rscLayer) cutRsc [QGVAR(title), "PLAIN", 0, false];
+    if (productVersion select 2 > 214) then { // TODO remove block after 2.16 release
+        (QGVAR(display) call BIS_fnc_rscLayer) cutRsc [QGVAR(title), "PLAIN", 0, false, false]; // draw under HUD
+    } else {
+        (QGVAR(display) call BIS_fnc_rscLayer) cutRsc [QGVAR(title), "PLAIN", 0, false];
```

Comment:
Might as well remove BIS_fnc_rscLayer? Can use string directly since 1.58.

Reply:
That's a separate PR.

Reply:
/shrug

Reply:
Also it was broken until 1.90. See the linked issue.